pulse_actions/worker.py

```diff
@@ -73,22 +74,41 @@ def load_config():
 
 def run_exchange_topic(topic_base):
     options = load_config()
+    topic_base = topic_base.split(",")
+    exchanges = []
+    topics = []
+    for topic in topic_base:
+        exchange = options[topic]['exchange']
+        exchanges.append(exchange)
+        topics.append(options[topic]['topic'])
     # Finding the right event handler for the given exchange and topic
     try:
-        handler_data = config.HANDLERS_BY_EXCHANGE[options[topic_base]['exchange']]
-        handler_function = handler_data[topic_base]
+        if len(topic_base) == 1:
+            handler_data = config.HANDLERS_BY_EXCHANGE[exchanges[0]]
+            handler_function = handler_data[topic_base[0]]
+        else:
+            handler_function = route_functions.route
     except KeyError:
         LOG.error("We don't have an event handler for %s with topic %s.",
-                  options[topic_base]['exchange'], options[topic_base]['topic'])
+                  exchanges, topics)
         exit(1)
 
-    run_pulse(
-        exchange=options[topic_base]['exchange'],
-        topic=options[topic_base]['topic'],
-        event_handler=handler_function,
-        topic_base=topic_base,
-        dry_run=False)
-
+    # We need to check if length of topic_base is 1, since pulse does not accept
+    # a list when there is only a single exchange/topic.
+    if len(topic_base) == 1:
+        run_pulse(
+            exchanges=exchanges[0],
+            topics=topics[0],
+            event_handler=handler_function,
+            topic_base=topic_base,
+            dry_run=False)
+    else:
+        run_pulse(
+            exchanges=exchanges,
+            topics=topics,
+            event_handler=handler_function,
+            topic_base=topic_base,
+            dry_run=False)
```

Comment:
I think we should make dry-run a command line argument, that way it will be easier to have some exchanges on and some exchanges off (I had to make something really hacky yesterday to keep automatic backfilling with dry-run and resultset_actions without dry-run)